README.md

```diff
@@ -1,95 +1,66 @@
+This repository hosts a work-in-progress Python implementation of the ICON climate and weather model. Additionally, it includes `icon4pytools`, a collection of command-line interfaces (CLIs), and utilities required for the integration of ICON4Py code into the ICON Fortran model. ICON4Py leverages [GT4Py](https://github.com/GridTools/gt4py) to ensure efficient and performance portable implementations of these components.
 
 ## Project Structure
 
-The repository is organized into directories, each containing independent Python namespace packages for different ICON components or utility packages. These packages can be installed independently. Since these packages are not available from a package repository (yet), you need to specify the location of dependencies within this repository. This can be done by installing the required dependencies first. Refer to the [Installation instructions](#installation-instructions) below.
+The repository is organized as a _monorepo_, where various ICON model components and utilities are developed as independent Python namespace packages in subfolders. An `icon4py` Python package is defined at the root folder just to collect the specific versions of the different components as project dependencies. The component packages can also be installed independently, although since they are not (yet) available from a package repository, they need to be installed from their specific location within this repository. This can be easily done by following the [Installation instructions](#installation-instructions) explained below.
 
 ## License
 
 ICON4Py is licensed under the terms of the BSD-3-Clause.
 
 ## Installation instructions
-### Dependencies
-A minimal installation of ICON4Py needs 
-- Python 3.10
-- boost >= 1.85.0
 
-You can install all packages at once by using the provided `requirements.txt` or `requirements-dev.txt` files in the root of the repository. For example:
-The `-dev.txt` file installs ICON4Py packages and GT4Py in editable mode, such that source changes are immediatly picked up and used in the virtual environment. 
-```bash
-# Clone the repository
-git clone git@github.com:C2SM/icon4py.git
-cd icon4py
-
-# Create a (Python 3.10) virtual environment (usually at `.venv`)
-python3.10 -m venv .venv
+Since this project is still in a highly experimental state, it is not yet available as a regular Python distribution project through PyPI. The expected installation procedure is to clone the [https://github.com/C2SM/icon4py](https://github.com/C2SM/icon4py) repository and install it in a venv using the following development workflow.  
```

Comment:
```suggestion
Since this project is still in a highly experimental state, it is not yet available as a regular Python distribution project through PyPI. The installation procedure comprises cloning the [https://github.com/C2SM/icon4py](https://github.com/C2SM/icon4py) repository and install it in a `venv` using the following development workflow.  
```